frontend/src/utils/fieldValidation.ts

```diff
@@ -78,30 +79,39 @@ type MinimumFieldValidationPropsEmailAndMobile<T extends FieldBase> = Omit<
 
 type ValidationRuleFn<T extends FieldBase = FieldBase> = (
   schema: MinimumFieldValidationProps<T>,
+  responseMode: FormResponseMode,
 ) => RegisterOptions
 
 type ValidationRuleFnEmailAndMobile<T extends FieldBase = FieldBase> = (
   schema: MinimumFieldValidationPropsEmailAndMobile<T>,
+  responseMode: FormResponseMode,
 ) => RegisterOptions
 
 const requiredSingleAnswerValidationFn =
-  (schema: Pick<FieldBase, 'required'>) => (value?: SingleAnswerValue) => {
-    if (!schema.required) return true
+  (schema: Pick<FieldBase, 'required'>, responseMode?: FormResponseMode) =>
+  (value?: SingleAnswerValue) => {
+    if (!schema.required || isMultirespondentForm(responseMode)) return true
```

Comment:
I'm struggling to figure out why `isMultirespondentForm(responseMode)` should `return true` here

Reply:
ah, `true` means that the value is accepted, while returning a string means the value is not accepted (with the string as the error message). 

So if the form is MRF, we always accept blank answers. Otherwise, the form is email or encrypt, in which case we only accept if the question is optional (i.e. `!schema.required`).